modules/conversion-center/assets/js/frontend/handlers/contact-buttons.js

```diff
@@ -72,6 +73,46 @@ export default class ContactButtonsHandler extends Base {
 		} );
 	}
 
+	contentWrapperIsHidden( hide ) {
+		if ( ! this.elements.contentWrapper ) {
+			return false;
+		}
+
+		const { hidden } = this.getSettings( 'constants' );
+
+		// Set current state
+		if ( true === hide ) {
+			this.elements.contentWrapper.classList.add( hidden );
+			this.elements.contentWrapper.setAttribute( 'aria-hidden', 'true' );
+			return;
+		}
+
+		if ( false === hide ) {
+			this.elements.contentWrapper.classList.remove( hidden );
+			this.elements.contentWrapper.setAttribute( 'aria-hidden', 'false' );
+			return;
+		}
+
+		// Get current state
+		return this.elements.contentWrapper.classList.contains( hidden );
```

Comment:
keeping the class based check as it is a faster check than getting the aria attribute state and it's a safe check since we're controlling these only in this function now